platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -211,13 +211,26 @@ void updateSep31Transaction(PatchTransactionRequest ptr, Sep31Transaction txn)
     validateTimestamps(txn);
   }
 
-  private void validatePlatformApiStatus(String status) throws BadRequestException {
+  /**
+   * validateIfStatusIsSupported will check if the provided string is a SepTransactionStatus
+   * supported by the PlatformAPI
+   *
+   * @param status a String representing the SepTransactionStatus
+   * @throws BadRequestException if the provided status is not supported
+   */
+  private void validateIfStatusIsSupported(String status) throws BadRequestException {
```

Comment:
We can make this package-private (same as the other validation methods) for unit-tests.

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/448/commits/85c61df17d3b6a4e3edd3c1bb9994b370174a982.